server/fleet/activities.go

```diff
@@ -1682,6 +1682,7 @@ type ActivityAddedAppStoreApp struct {
 	TeamName      *string             `json:"team_name"`
 	TeamID        *uint               `json:"team_id"`
 	Platform      AppleDevicePlatform `json:"platform"`
+	SelfService   bool                `json:"self_service"`
```

Comment:
This new field needs to be added to the `Documentation` method, and the docs need to be re-generated (`make generate-doc`).

Reply:
The `Documentation` method is just below that line, it documents the fields of the activity. Once that is modified, `make generate-doc` will have changes.